lib/controllers/dojo/save.spec.js

```diff
@@ -96,6 +97,82 @@ lab.experiment('save a dojo', function () {
       done();
     });
   });
+  // TODO : test not changing upon equality
+  lab.describe('the events should be notified', function () {
+    var l_dojo;
```

Comment:
`l_dojo`?

Reply:
local_dojo.
It's because we make a copy because.. seneca inmemory modifies.. in memory so what you pass is directly modified.